buildscripts/incremental/test.sh

```diff
@@ -21,6 +21,10 @@ export NUMBA_DEVELOPER_MODE=1
 # enable the fault handler
 export PYTHONFAULTHANDLER=1
 
+if [ "$TEST_TBB" == "yes" ]; then
+export NUMBA_THREADING_LAYER='tbbpool'
```

Comment:
can we change it to `NUMBA_THREADING_LAYER=TBB` so that it is closer to how users would like to see it (and they might be learning from this line as well)

Reply:
I think this needs adjustment anyway as it was in early on before e.g. omp was added, happy to accommodate this.